.travis.yml

```diff
@@ -18,6 +18,7 @@ before_install:
   # extralibs
   - wget https://github.com/ericsun2/sandbox/raw/master/extralibs/extralibs.zip
   - mkdir -p metadata-etl/extralibs; unzip extralibs.zip -d metadata-etl/extralibs
+  - mkdir -p RestLiClient/extralibs; unzip extralibs.zip -d RestLiClient/extralibs
```

Comment:
Later we may want to separate the extraLib into two zips, one for each component. Currently it's copying all jars to both locations.